README.rst

```diff
@@ -446,26 +464,69 @@ code by importing the base admin classes and registering your models with.
 
     # admin.py of your app
     from django.contrib import admin
-
     from django_x509.base.admin import CaAdmin as BaseCaAdmin
     from django_x509.base.admin import CertAdmin as BaseCertAdmin
-
     from .models import Ca, Cert
 
-
     class CaAdmin(BaseCaAdmin):
         # extend/modify the default behaviour here
         pass
 
-
     class CertAdmin(BaseCertAdmin):
         # extend/modify the default behaviour here
         pass
 
-
     admin.site.register(Ca, CaAdmin)
     admin.site.register(Cert, CertAdmin)
 
+Extending ``apps.py``
+~~~~~~~~~~~~~~~~~~~~~
+
+Extending django-x509 app config is simple, you can
+use the following:
+
+.. code-block:: python
+
+    from django_x509.apps import DjangoX509Config
+
+    class SampleX509Config(DjangoX509Config):
+        name = 'MY_AWESOME_APP'
+        verbose_name = 'MY_AWESOME_APP'
+
+
+Extending tests
+~~~~~~~~~~~~~~~
+
+If you want to extend the testcases from django-x509:
+
+.. code-block:: python
+
+    import os
```

Comment:
here's where we may want to do it as in https://github.com/openwisp/openwisp-firmware-upgrader/pull/60/files#diff-40eae13e16e4642fe9c9151affe297b2L1-R55